_includes/docs/extending/configuring.md

```diff
@@ -0,0 +1,14 @@
+Configuration options for modules can be in many places.
+The most common places to look for are the `Extend` and the `Configuration` page.
+Both can be accessed via the menu at the top.  
+Also the documentation of the module on [http://drupal.org]() should point to the correct place.
+Note, that not all modules have configuration pages.
+
+### Extend page
+Go to the `Extend` page and search for your module.
+If the module is configurable and the maintainer included the correct route in his module,
+you can find a `Configure` link, after expanding the module description, by clicking on it.
```

Comment:
I don't get this: How can I find the link by clicking on it? Doesn't sound right.